public/app/core/components/AppChrome/AppChrome.tsx

```diff
@@ -25,17 +27,24 @@ export interface Props extends PropsWithChildren<{}> {}
 
 export function AppChrome({ children }: Props) {
   const { chrome } = useGrafana();
+  const { isOpen: isExtensionSidebarOpen } = config.featureToggles.extensionSidebar
+    ? useExtensionSidebarContext()
+    : { isOpen: false };
```

Comment:
wonder if we can move the  config.featureToggles.extensionSidebar  check useExtensionSidebarContext? 

maybe have a isEnabled property on the context?

Reply:
Done in e4b8ce7497209bea72f4b1cc8248aeaec8abfed6